plugins/modules/route53_health_check.py

```diff
@@ -632,17 +632,17 @@ def main():
             # If health_check_name is a unique identifier
             if module.params.get('use_unique_names'):
                 existing_checks_with_name = get_existing_checks_with_name()
+                if tags is None:
+                    module.params['purge_tags'] = False
```

Comment:
I do not think we can edit the module.params here. It is better to check for the value of purge_tags and do the necessary action.

Reply:
I've tried making changes according to discussion earlier on slack
https://github.com/ansible-collections/amazon.aws/pull/1253/files#r1024771803
Although, open to new ideas :)

Reply:
I am unsure if anyone has suggested making changes to the value of purge_tags.  Correct me if I am wrong.

Reply:
@GomathiselviS @goneri fixed. correct? 
https://github.com/ansible-collections/amazon.aws/pull/1253/commits/d2150481498351bf8802daa4635fd34b4303197d